### PR TITLE
Added default FeedBuilder

### PR DESCRIPTION
**Pull request by valdisiljuconoks**
Default FeedBuilder should be used if you want to be lazy and not do error handling or catalog entry enumeration.

### CODE REVIEW

**Comment by valdisiljuconoks** [comment]
Also added reference to Episerver.Commerce package

File: src/Geta.GoogleProductFeed/DefaultFeedBuilderBase.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            var generatedFeeds = new List<Feed>();
+
+            var feed = GenerateFeedEntity() ?? throw new ArgumentNullException($"{nameof(GenerateFeedEntity)} returned null");
+
+            var entries = new List<Entry>();
+            var catalogReferences = _contentLoader.GetDescendents(_referenceConverter.GetRootLink());
+
+            foreach (var catalogReference in catalogReferences)
+            {
+                var catalogContent = _contentLoader.Get<CatalogContentBase>(catalogReference);
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/DefaultFeedBuilderBase.cs]: It's better to batch load catalog content items using _contentLoader.Items()

File: src/Geta.GoogleProductFeed/DefaultFeedBuilderBase.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+using EPiServer.Logging;
+using Geta.GoogleProductFeed.Models;
+using Mediachase.Commerce.Catalog;
+
+namespace Geta.GoogleProductFeed
+{
+    public abstract class DefaultFeedBuilderBase : FeedBuilder
+    {
+        // ReSharper disable once MemberCanBePrivate.Global
+        protected readonly IContentLoader _contentLoader;
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/DefaultFeedBuilderBase.cs]: Why not private?

File: src/Geta.GoogleProductFeed/FeedHelper.cs [hunk 3]
```diff
@@ -26,7 +25,7 @@ public FeedHelper(FeedBuilder feedBuilder, IFeedRepository feedRepository)
 
         public bool GenerateAndSaveData()
         {
-            List<Feed> feeds = _feedBuilder.Build();
+            var feeds = _feedBuilder.Build();
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/FeedHelper.cs]: Why? With var you have to hover over to see what the return will be. What was was more readable

**Pull request merged**